### DISCUSSION TITLE
My website looks cheap:-)

### DISCUSSION THREAD

**Discussion by zeframek** [masterking32/WoWSimpleRegistration#90] [Q&A] [answered]
![Snímek obrazovky 2023-11-28 235300](https://github.com/masterking32/WoWSimpleRegistration/assets/24695495/34e25c52-8656-416c-a4db-2a11aa4fbf79)
Hello, everybody,
my site looks like without image and styles can anyone advise what to do?  Thank you very much in advance.

**Comment by masterking32** [comment, marked as answer]
Hello, check this: https://github.com/masterking32/WoWSimpleRegistration/issues/32 and https://github.com/masterking32/WoWSimpleRegistration/issues/38
